package.json

```diff
@@ -57,6 +57,7 @@
     "@backstage/create-app": "workspace:*",
     "@backstage/repo-tools": "workspace:*",
     "@changesets/cli": "^2.14.0",
+    "@frsource/cypress-plugin-visual-regression-diff": "^3.2.4",
```

Comment:
This needs to be added to the `package.json` in `packages/techdocs-cli` and not here:
```bash
cd packages/techdocs-cli
yarn add -D @frsource/cypress-plugin-visual-regression-diff
```
In that file, you'll also find the line `cypress-plugin-snapshots": "^1.4.4`, which needs to be removed.